client/state/data-layer/wpcom-http/actions.js

```diff
@@ -21,6 +22,7 @@ import { WPCOM_HTTP_REQUEST } from 'state/action-types';
  */
 export const http = ( {
 	apiVersion = 'v1',
+	apiNamespace = 'wpcom/v2',
```

Comment:
@retrofox I'm pretty sure this default caused the regression for the plans endpoint. Let's be sure to update the test plan for the next attempt to include smoke testing plans/signup flow